packages/patternfly-react/less/compound-label.less

```diff
@@ -0,0 +1,42 @@
+@import 'patternfly/variables';
+
+.pf-remove-button {
```

Comment:
These selectors should have a `compound-label-pf` or similar prefix so as not to mistakenly override.